cmd/minikube/cmd/start.go

```diff
@@ -1314,6 +1318,24 @@ func validateListenAddress(listenAddr string) {
 	}
 }
 
+// This function validates that the --ports are not below 1024 for the host and not outside range
+func validatePorts(ports []string) {
+	for _, portDuplet := range ports {
+		for i, port := range strings.Split(portDuplet, ":") {
+			p, err := strconv.Atoi(port)
+			if err != nil {
+				exit.Message(reason.Usage, "Sorry, one of the ports provided with --ports flag is not valid {{.ports}}", out.V{"ports": ports})
```

Comment:
how about making this return fmt.Errorf("error string...")
and then Exit in the caller with the error message

and then we could add a unit test for

validatePort in the start_test.go

Reply:
I just followed the "style" of the rest of validation functions but this makes total sense